package.json

```diff
@@ -25,11 +26,12 @@
     "link-all": "node scripts/link-all.js packages",
     "unlink-all": "node scripts/unlink-all.js packages",
     "check": "flow check",
+    "flow": "lerna run build-ts && flow",
     "lint": "eslint . && prettier \"./packages/*/*/{src,bin,test}/**/*.{js,json,md}\" --list-different && cargo fmt --all -- --check",
     "prepublishOnly": "yarn adjust-versions && yarn build && yarn build-ts",
-    "test:unit": "cross-env NODE_ENV=test mocha --timeout 5000 && cargo test",
-    "test:integration": "yarn workspace @parcel/integration-tests test",
-    "test:integration-ci": "yarn workspace @parcel/integration-tests test-ci",
+    "test:unit": "lerna run build-ts && cross-env NODE_ENV=test mocha --timeout 5000 && cargo test",
```

Comment:
You should be able to remove the explicit `build-ts` (which is quite heavy handed) by modifying the parcel babel preset

Reply:
I'd rather shift towards split compile and run lifecycle than the current mix of babel register etc.

It's quite confusing at the moment that there is the "dev" register mode and another mode that is pre-built (hence need the dev-prepare steps to fix-up the repository).